docs/variants.md

```diff
@@ -56,7 +59,11 @@ Possible variants are:
 ### Fragmentation
 
 ### Missing manuscripts
+In the StemmaBench package, the concept of missing manuscripts is modeled using a probabilistic process governed by a p-parameter Bernoulli distribution. After having generating the whole tradition, a proportion *p* of the manuscripts are selected (with equiprobability) and deleted from the tradition. This simulates the loss or absence of certain manuscripts over time.
```

Comment:
I think simply mentioning "loss" instead of "loss or absence" is enough.

Reply:
- [x] "loss or absence" replace by "loss".